3DFin.yaml

```diff
@@ -0,0 +1,42 @@
+
+# Partially generated by req2flatpak.py --requirements-file somefile.txt --target-platforms 310-x86_64 --yaml --outfile=dendromatics.yaml
+# some transitive native dependencies were fetched manually. 
+name: python3-3DFin-deps
+buildsystem: simple
+build-commands:
+- pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+  --prefix=${FLATPAK_DEST} --no-build-isolation python_dateutil pytz tzdata dendromatics lazrs pandas pydantic XlsxWriter 3DFin
+sources:
+- type: file
+  url: https://files.pythonhosted.org/packages/57/c2/6a6e7b46226c5733ec3dc54abfa1dad130b8a8c0c3365fc0502585027d98/dendromatics-0.2.0-py3-none-any.whl
+  sha256: 60e179efd830b779d0ddc5c590cd61d754beadd6d56af666595150999d21093b
+- type: file
+  url: https://files.pythonhosted.org/packages/d1/7f/0eac494e5b87d313a19e19bd8a0c9d012b60c7e042acd8a9b55af251d32f/lazrs-0.5.3-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+  sha256: 85899e3c6c7af5989d8bfe1446515e8290794b423a0dd3f237dc5d0b3038300a
+  only-arches:
+  - x86_64
+- type: file
+  url: https://files.pythonhosted.org/packages/d0/28/88b81881c056376254618fad622a5e94b5126db8c61157ea1910cd1c040a/pandas-2.0.3-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+  sha256: d9cd88488cceb7635aebb84809d087468eb33551097d600c6dad13602029c2df
+  only-arches:
+  - x86_64
+- type: file
+  url: https://files.pythonhosted.org/packages/d2/16/2edfe3e52de9d46fee81d9b9ace90fd7a49a86e7a36d7fc280183f77515a/pydantic-1.10.13-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+  sha256: e31647d85a2013d926ce60b84f9dd5300d44535a9941fe825dc349ae1f760df9
+  only-arches:
+  - x86_64
+- type: file
+  url: https://files.pythonhosted.org/packages/bd/00/28d48d105fa914b460e9721b315af096937e2e82bf540932c68420ccd9e1/XlsxWriter-3.1.0-py3-none-any.whl
+  sha256: b70a147d36235d1ee835cfd037396f789db1f76740a0e5c917d54137169341de
+- type: file
+  url: https://files.pythonhosted.org/packages/36/7a/87837f39d0296e723bb9b62bbb257d0355c7f6128853c78955f57342a56d/python_dateutil-2.8.2-py2.py3-none-any.whl
+  sha256: 961d03dc3453ebbc59dbdea9e4e11c5651520a876d0f4db161e8674aae935da9
+- type: file
+  url: https://files.pythonhosted.org/packages/4f/a4/879454d49688e2fad93e59d7d4efda580b783c745fd2ec2a3adf87b0808d/pytz-2020.1-py2.py3-none-any.whl
+  sha256: a494d53b6d39c3c6e44c3bec237336e14305e4f29bbf800b599253057fbb79ed
+- type: file
+  url: https://files.pythonhosted.org/packages/89/2d/49329ebec33b14dae61ecc8c85abe596341832fa36c4bcd3d99fddda018b/tzdata-2022.1-py2.py3-none-any.whl
+  sha256: 238e70234214138ed7b4e8a0fab0e5e13872edab3be586ab8198c407620e2ab9
+- type: file
+  url: https://files.pythonhosted.org/packages/85/d5/ab95fff354793b25019416b41fd75d3b2f71040902b2a4d1378c86413614/3dfin-0.2.0rc4-py3-none-any.whl
+  sha256: e7fd2b14ce025a59311d3b18fdd0f38f03a095e9fa8e5317e055b361f99ce812
```

Comment:
Here, and in file below, there is a new line missing at the end of the last line (EOL). Can you make sure there is a EOL in every text file?